src/pages/Map/components/WeatherMarker/WeatherMarker.tsx

```diff
@@ -49,7 +49,7 @@ const WeatherMarker = ({ pos, cityName }: Props) => {
       state={{ cityName }}
     >
       <Marker icon={DefaultIcon} position={pos} opacity={0}>
-        <Content interactive={true} permanent={true} direction="top">
+        <Content opacity={1} interactive={true} permanent={true} direction="top">
```

Comment:
**question** why opacity is passed as a prop?

Reply:
@kamil-borecki-cc 
Because without this props Tooltip has default opacity 0.9 as far as I remember, so it also changes opacity for each nested elements which we do not want.

And also if I change background opacity without providing this opacity prop that you are asking about, we can't make background opacity 100% because Tooltip Marker has opacity 0.9 and it will be still 0.9 transparency of tooltip even on hover.

That's why I had to change it to `opacity={1}`

Reply:
Why can't it be done through CSS?

Reply:
@kamil-borecki-cc 
Because of this awesome react-leaflet library :D 
It has some built-in features that doesn't allow us to do it manually in CSS, unless i do it adding `!important` after opacity style.
What do you prefer?
Or maybe you have another solution for this? :)

Reply:
Ok, let's leave it as it is, thanks for the clarification.